### ISSUE TITLE
Any way to show a framerate graph rather than frame time?

### ISSUE THREAD

**Issue by Mossy93** [flightlessmango/MangoHud#738]
Hello,

I would like to show a graph of the framerate rather than the frame time. Is there any way to do this? I know this is something you can do on Windows with MSI Afterburner.

Thank you.

**Comment by the-science-guy** [comment]
![image](https://user-images.githubusercontent.com/98602834/166803311-fc85ce36-fad0-459e-8f42-04c1cdac268b.png)
You mean like this?

just add

frame_timing

to your ~.config/MangoHud/MangoHud.conf

**Comment by Mossy93** [comment]
Yes, but that is frame time. I wanted the frame rate in frames per second as a graph. I don't think I saw that functionality.

**Comment by the-science-guy** [comment]
> Yes, but that is frame time. I wanted the frame rate in frames per second as a graph. I don't think I saw that functionality.

Oh, pardon me, I misunderstood your request then, my apologies. AFAIK it is not implemented (yet).

**Comment by Mossy93** [comment]
You're okay. Keep up the great work! 👍

**Comment by shmerl** [comment]
I couldn't find additional info on that, but is there some way for any parameter to be shown as a graph and not as a number or currently it's only limited to frametime?

**Comment by shmerl** [comment]
Any news on this?

**Comment by BETLOG** [comment]
Bump because I have wanted to do this for as long as I can remember, and not being able to means each time i reinstall mangohud to see if its useful to me i usually end up uninstalling it a few days later because it does not include the #1 thing I want to display: an FPS graph.
When has an actual gamer ever been heard quoting frametime?
Note the linegraph here; have they *actually* resorted to using multiple huds at once just to get the typical linegraphs that we normally want to be able to compare?
Can anyone tell me how is this done? Gallium maybe?
![2024-11-19--18-11-40_betlogbeast_1920x1080+0+0](https://github.com/user-attachments/assets/c7d24849-dc83-4cca-8aba-10672c3f4e18)
